cdap-ui/app/features/admin/templates/namespace/create.html

```diff
@@ -24,6 +27,7 @@
              ng-model="model.name"
              class="form-control"
              placeholder="Name"
+             required
```

Comment:
We use name to display in the UI. I think that would be necessary for us to display it in the UI.